questionnaire/submissions-routes.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const express = require('express');
+const validateJWT = require('express-jwt');
+
+const createSubmissionsService = require('./submissions-service');
+const permissions = require('../middleware/route-permissions');
+
+const router = express.Router();
+
+router.use(validateJWT({secret: process.env.DCS_JWT_SECRET}));
+
+router
+    .route('/resubmit-failed')
+    .post(permissions('update:questionnaires'), async (req, res, next) => {
+        try {
+            const submissionsService = createSubmissionsService({logger: req.log});
+            submissionsService.postSubmissionsByStatus('failed');
```

Comment:
Given that we now have a distinct submission function and a resubmission function and we shouldn't be able to fall into any unwanted/unexpected statuses, could we not just lock this to only find failed applications? Or are you thinking this still needs to be reusable for other statuses?